src/ui/Table/Paged.svelte

```diff
@@ -44,83 +76,85 @@
   function onNextPage() {
     if (page >= pagesAmount) page = maxPage
     else page++
-    onPageChange(page)
+    onPageChange(page, pageSize)
   }
 
   function onPrevPage() {
     if (page <= 1) page = 0
     else page--
-    onPageChange(page)
+    onPageChange(page, pageSize)
   }
+
+  onMount(() => {
+    if (hasMoreItems) {
+      onPageChange(0, pageSize)
+    }
+  })
```

Comment:
`onPageChange` should be called only when page change is manually triggered. Such behaviour could be added by a library user